### ISSUE TITLE
slider-container expands more than the visible slides

### ISSUE THREAD

**Issue by alexberendei** [Wikiki/bulma-carousel#86]
![Screenshot from 2019-07-02 10-05-20](https://user-images.githubusercontent.com/677755/60491367-4f97be80-9cb1-11e9-82be-c520c6a6b894.png)

I have this behavior after I've updated the bulma carousel to 4.0.4. 

My quick fix was to override the css "slider" class and add `overflow:hidden `

**Comment by tehfink** [comment]
I had the same issue, so thanks for the tip. Unfortunately, it seems to break the pagination as well…

**Comment by manushichev** [comment]
@tehfink yes, navigation stops working with `overflow:hidden`. And not immediately, but after some actions, the logic of which I can not understand.
Probably, a quick press "right" and "left" among others causes a breakdown. Also, the transition from dot navigation to arrows often causes a crash.

**Comment by eman1986** [comment]
I hope this gets fixed soon, the temp fix posted works for now for me

**Comment by tiglapiles** [comment]
I hava the same problem. It's fine in firefox and sarafi, just in chrome it expends the visible slides in device mode.

![image](https://user-images.githubusercontent.com/26108622/66632899-1ec60e80-ec3c-11e9-9fe0-73b2e8f1a7d9.png)

**Comment by tiglapiles** [comment]
this is the sass source code,why comment overflow?
![image](https://user-images.githubusercontent.com/26108622/66638383-6a31ea00-ec47-11e9-988e-f45853db1b1d.png)

**Comment by tiglapiles** [comment]
there is the alternative plan use react: https://www.npmjs.com/package/react-responsive-carousel

**Comment by eman1986** [comment]
some of us don't use react or ever want to.

**Comment by tiglapiles** [comment]
year but bulma extensions's js is sock.

**Comment by Sykat** [comment]
Adding `overflow:hidden` in Container worked for me.
You can also add _is-clipped_ class in container
https://codepen.io/adhaxious/pen/JjdRwvO
